repos/fdbt-site/src/components/RadioConditionalInput.tsx

```diff
@@ -367,7 +368,7 @@ const renderConditionalRadioButton = (
             type="radio"
             value={radio.value}
             data-aria-controls={radio.dataAriaControls}
-            defaultChecked
+            defaultChecked={radio.defaultChecked}
```

Comment:
As above,this one is *always* checked, so we presumably shouldn't be changing it so its sometimes checked?